### ISSUE TITLE
Table/Grid outline is not visible on the button

### ISSUE THREAD

**Issue by kolaps33** [microsoft/fluentui#16764]
Repro steps:
1.
Open example: Nested navigation in table by row and cell
https://fluentsite.z22.web.core.windows.net/0.52.0/components/table/definition#usage-navigable
2. Navigate to "..." button in the table/grid
3. Observe outline

Actual: outline is not visible, pl. see screenshot below
Expected: outline should be visible

![image](https://user-images.githubusercontent.com/22620150/106594495-76d00500-6552-11eb-91b5-a5bb88bf1345.png)

**Comment by joschect** [comment]
@kolaps33 Thanks for the submission! This looks like this is in reference to the Stardust table, is that correct?

**Comment by gouttierre** [comment]
@kolaps33 : Thanks for filling this item. I do see the more options button does not have an outline.  I’ll mark this as an accessibility request and add it to our backlog. 

cc @smhigley
